### PR TITLE
Ablility to update all settings with a Map

### PR DESCRIPTION
**Pull request by dariosilva**
Since we can have multiples options to configure the generation of pdfs, I think that is useful to make a Map structure with the settings a pass it to the classes: HtmlToPdf and HtmlToPdfObject.

This way we can load a bunch of settings (e.g. from external properties files) in a few lines of code.

So, I propose to add two new methods to allow the update the settings with a Map.

### CODE REVIEW

**Review by benbarkay — CHANGES_REQUESTED** [review]
Thank you for the PR, I will gladly merge once the changes are made :)

File: src/main/java/io/woo/htmltopdf/HtmlToPdf.java [hunk 1]
```diff
@@ -282,6 +282,11 @@ private HtmlToPdf setting(String name, String value) {
         return this;
     }
 
+    public HtmlToPdf setting(Map<String,String> settings) {
```
[the remark this answers is not in the document]
Comment (dariosilva) [reply src/main/java/io/woo/htmltopdf/HtmlToPdf.java]: Yes, you are right! I made the changes in the new commit: https://github.com/wooio/htmltopdf-java/pull/16/commits/b31d8570a7cff7567c4417ab93bcff8a9222cf27#diff-84732fa7114c570fa208a481f5fabc86R31

File: src/main/java/io/woo/htmltopdf/HtmlToPdf.java [hunk 2]
```diff
@@ -282,6 +282,11 @@ private HtmlToPdf setting(String name, String value) {
...
+        this.settings = settings;
```
[the remark this answers is not in the document]
Comment (dariosilva) [reply src/main/java/io/woo/htmltopdf/HtmlToPdf.java]: Solved. Many thanks.

File: src/main/java/io/woo/htmltopdf/HtmlToPdfObject.java [hunk 3]
```diff
@@ -306,6 +306,11 @@ private HtmlToPdfObject setting(String name, String value) {
         return this;
     }
 
+    public HtmlToPdfObject setting(Map<String,String> settings) {
```
[the remark this answers is not in the document]
Comment (dariosilva) [reply src/main/java/io/woo/htmltopdf/HtmlToPdfObject.java]: Changes made at the new commit: https://github.com/wooio/htmltopdf-java/pull/16/commits/b31d8570a7cff7567c4417ab93bcff8a9222cf27#diff-a539f6324aadc3006396f439efd0d005R24 and https://github.com/wooio/htmltopdf-java/pull/16/commits/b31d8570a7cff7567c4417ab93bcff8a9222cf27#diff-a539f6324aadc3006396f439efd0d005R48

**Review by benbarkay — APPROVED** [review]
There are still a couple of things that bug me, but I'll change them myself before releasing.
Thank you for your contribution :)

**Comment by dariosilva** [comment]
Thank you @benbarkay :)

**Pull request merged**